doc/stdenv/stdenv.chapter.md

```diff
@@ -990,6 +990,22 @@ Convenience function for `makeWrapper` that replaces `<\executable\>` with a wra
 
 If you will apply it multiple times, it will overwrite the wrapper file and you will end up with double wrapping, which should be avoided.
 
+### `prependToVar` \<variableToPrependTo\> \<elements\> {#fun-prependToVar}
+
+Prepends elements to a variable. For example:
```

Comment:
> , separated by spaces.

I don't think this is right. They're supposed to be separate arguments.
Space separation is an implementation detail of the bash syntax, which is lower level than what we should discuss here, to avoid confusion.
For context, it is also not a complete description, because newlines are also valid argument separators:

```
prependToVar buildInputs \
  foo \
  bar
```

But enough about that.

```suggestion
Prepend elements to an array variable.

Example:
```

(or does it not have to be an array at first?)

Reply:
My comment was about what the result looks like. The phrasing may not have made that clear.

How the elements are passed should be noted as well though, as you suggested with `...`. These two are independent.

Reply:
> (or does it not have to be an array at first?)

both strings and arrays work